cloud/shared/bin/deploy

```diff
@@ -12,4 +12,10 @@ if [[ "${IMAGE_TAG}" == "latest" ]]; then
 fi
 
 export TF_VAR_image_tag="${IMAGE_TAG}"
-exec "cloud/${CIVIFORM_CLOUD_PROVIDER}/bin/deploy" "$@"
+if [[ -f "cloud/${CIVIFORM_CLOUD_PROVIDER}/bin/deploy.py" ]]; then
```

Comment:
Is this fork necessary? Post-this PR, the Python script will always be present. I'd suggest also deleting the bash version here as well, since it wouldn't need to be used anymore.

Reply:
this is shared script, it is invoked for both azure and aws. Added todo to cleanup.

Reply:
Gotcha. Thanks for the context :)